src/components/message/Message.jsx

```diff
@@ -64,16 +64,16 @@ export default class MessageContainer extends React.Component {
   }
 
   render() {
-    const { side, type, sender, ...props } = this.props;
+    const { side, type, sender, className, ...props } = this.props;
     const Component = componentsDict[type];
     const disableBubble = ['actions', 'botAction', 'cards'].includes(type);
 
     return disableBubble ? (
       <Component type={type} {...props} />
     ) : (
-      <ClearDiv component={Component}>
-        {side === 'left' && <Avatar sender={sender} />}
-        <Bubble side={side} isImage={type === 'image'}>
+      <ClearDiv className={`bf-message-container ${sender} ${className}`} component={Component}>
```

Comment:
`sender` is simply `user` or `bot` so this might not be specific enough, maybe add bf so it becomes `bf-${sender}` or something